intel_extension_for_transformers/neural_chat/pipeline/plugins/retrieval/detector/intent_detection.py

```diff
@@ -18,7 +18,7 @@
 
 from intel_extension_for_transformers.neural_chat.pipeline.plugins.prompt.prompt_template \
     import generate_intent_prompt
-from intel_extension_for_transformers.llm.inference import predict
+from intel_extension_for_transformers.neural_chat.utils.model_utils import predict

```

Comment:
predict/predict_stream is better in inference?

Reply:
is changing model_utils to inference_utils ok?